android/src/main/java/com/azure/reactnative/notificationhub/ReactNativeNotificationHubModule.java

```diff
@@ -377,6 +386,54 @@ public void isNotificationEnabledOnOSLevel(Promise promise) {
         promise.resolve(areNotificationsEnabled);
     }
 
+    @ReactMethod
+    public void scheduleLocalNotification(ReadableMap map, String timeMsString, Promise promise) {
```

Comment:
This is the only "new" feature right? Besides `RCTConvert` for IOS. The rest of the edits are more refactorings. Correct?

Reply:
Correct